pkg/acsengine/const.go

```diff
@@ -1,44 +1,8 @@
 package acsengine
 
 const (
-	// DefaultOpenShiftMasterSubnet is the default value for master subnet for Openshift.
-	DefaultOpenShiftMasterSubnet = "10.0.0.0/24"
-	// DefaultOpenShiftFirstConsecutiveStaticIP is the default static ip address for master 0 for Openshift.
-	DefaultOpenShiftFirstConsecutiveStaticIP = "10.0.0.11"
-	// DefaultMasterSubnet specifies the default master subnet for DCOS or Swarm
-	DefaultMasterSubnet = "172.16.0.0/24"
-	// DefaultFirstConsecutiveStaticIP specifies the static IP address on master 0 for DCOS or Swarm
-	DefaultFirstConsecutiveStaticIP = "172.16.0.5"
-	// DefaultSwarmWindowsMasterSubnet specifies the default master subnet for a Swarm Windows cluster
-	DefaultSwarmWindowsMasterSubnet = "192.168.255.0/24"
-	// DefaultSwarmWindowsFirstConsecutiveStaticIP specifies the static IP address on master 0 for a Swarm WIndows cluster
-	DefaultSwarmWindowsFirstConsecutiveStaticIP = "192.168.255.5"
-	// DefaultDCOSMasterSubnet specifies the default master subnet for a DCOS cluster
-	DefaultDCOSMasterSubnet = "192.168.255.0/24"
-	// DefaultDCOSFirstConsecutiveStaticIP  specifies the static IP address on master 0 for a DCOS cluster
-	DefaultDCOSFirstConsecutiveStaticIP = "192.168.255.5"
-	// DefaultDCOSBootstrapStaticIP specifies the static IP address on bootstrap for a DCOS cluster
-	DefaultDCOSBootstrapStaticIP = "192.168.255.240"
-	// DefaultKubernetesMasterSubnet specifies the default subnet for masters and agents.
-	// Except when master VMSS is used, this specifies the default subnet for masters.
-	DefaultKubernetesMasterSubnet = "10.240.0.0/16"
-	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
-	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
```

Comment:
Did this const get moved or just deleted?

Reply:
These have been moved to package api.